lucet-validate/tests/wasitests.rs

```diff
@@ -57,7 +56,7 @@ mod lucet_validate_tests {
             };
             validator
                 .validate(&entry_wasm)
-                .expect(&format!("validate {:?}", entry_path));
+                .unwrap_or_else(|_| panic!("validate {:?}", entry_path));
```

Comment:
I didn't realize `expect` doesn't take a format-like series of arguments. huh! This undoubtedly would have been formatting the error string even on non-errors so :+1: